src/marqo/tensor_search/models/api_models.py

```diff
@@ -284,6 +286,10 @@ def get_context_tensor(self) -> Optional[List[SearchContextTensor]]:
         """Extract the tensor from the context, if provided"""
         return self.context.tensor if self.context is not None else None
 
+    def get_context_documents(self) -> Optional[SearchContextDocuments]:
+        """Extract the documents from the context, if provided"""
+        return self.context.documents if self.context is not None else None
```

Comment:
just `return self.context.documents`?

Reply:
But this will break if `self.context = None`